diag_manager/fms_diag_yaml.F90

```diff
@@ -30,7 +30,7 @@
 !> @{
 module fms_diag_yaml_mod
 #ifdef use_yaml
-use diag_data_mod,   only: DIAG_NULL
+use diag_data_mod,   only: DIAG_NULL, DIAG_OCEAN, DIAG_ALL, DIAG_OTHER
```

Comment:
do these already exist, or did you add them?

Reply:
These were already in diag_data_mod